### DISCUSSION TITLE
🐛 BUG: Disable useless emmet abbreviation when trying to reference variables from script.

### DISCUSSION THREAD

**Discussion by jamesmcl113** [withastro/language-tools#835] [Feature requests]
### Describe the Bug

I've setup the Astro LS and it's working nicely. However, when writing JSX in a `.astro` file I get an annoying emmet abbreviation completion when I'm trying to reference a name from the script section of the file. Image below shows the completion:
<img width="747" alt="Screenshot 2024-03-16 at 18 37 13" src="https://github.com/withastro/language-tools/assets/29102582/689082af-da60-48c2-ad74-b3a4baa9233b">

If I select the completion, it just pastes in the name without curly braces. It's pretty annoying since it's always the first completion that appears in the list. I'd like to disable this without disabling emmet for the whole language server if possible.

### Steps to Reproduce

1. `npm init astro` using template <NAME>
2. Setup astro-ls for neovim
3. Create a `.astro` file with a variable in the script section
4. Begin typing name from the script section in HTML + JS section
5. Annoying emmet abbreviation auto-complete options appears

**Comment by Princesseuh** [comment]
I agree! Emmet doesn't really like being disabled partially, but this problem has annoyed me in the past too, so definitely need to figure out a solution.

**Comment by IcyFoxe** [comment]
I'm also hoping for this issue to be resolved in some way without disabling Emmet completely.

**Comment by Princesseuh** [comment]
The last update of the extensions should improve the situation. It's still not quite like it is in JSX, not sure why yet, but it seems generally better

**Reply by Princesseuh** [reply]
... Okay, so it does improve on the situation, but I just noticed it brings on other things that we don't want 🤦‍♀️ so nevermind that, will revert
